### ISSUE TITLE
About dummy keypoints for batch mode

### ISSUE THREAD

**Issue by bakisanlan** [cvg/LightGlue#158]
Hello everyone,

I try to use LightGlue in batch mode for getting faster results. Query images should have same number of keypoints if batch mode is used. Hence, I try to create dummy keypoints for resolving that issue. First, I try to find match between image1(2048 keypoints) and image2 (1050 keypoints) without batch mode. Then, I compare the non-batch matching results with batched matching results. Below I shared the code how I create dummy keypoints and adding to feature dict. 

```
# matching before dummy keypoints
matches01 = matcher({"image0": feats0_org, "image1": feats1_org})

# matching after padding with dummy keypoints
n_pad = abs(feats0["keypoints"].shape[0] - feats1["keypoints"].shape[0])

# automatically select features to be padding
feat_willbe_padded = feats0_org.copy()
if feats0_org["keypoints"].shape[1] > feats1_org["keypoints"].shape[1]:
    feat_willbe_padded = feats1_org.copy()
    
pad_kp    =   torch.zeros((feat_willbe_padded["keypoints"].shape[0], n_pad, 2)).to(feat_willbe_padded["keypoints"].device)
pad_desc  =  torch.zeros((feat_willbe_padded["descriptors"].shape[0], n_pad, 256)).to(feat_willbe_padded["descriptors"].device)
pad_score = torch.zeros((feat_willbe_padded["keypoint_scores"].shape[0], n_pad)).to(feat_willbe_padded["keypoint_scores"].device)

feat_willbe_padded["keypoints"]          = torch.cat([feat_willbe_padded["keypoints"], pad_kp], dim=1)
feat_willbe_padded["descriptors"]        = torch.cat([feat_willbe_padded["descriptors"], pad_desc], dim=1) 
feat_willbe_padded["keypoint_scores"] = torch.cat([feat_willbe_padded["keypoint_scores"], pad_score], dim=1)

matches02 = matcher({"image0": feats0_org, "image1": feat_willbe_padded})
```
I observe that, the number of matched keypoints nearly zero if I match with dummy keypoints while nearly 200 keypoints is found without dummy keypoints . Do you have any idea why it is happening or do you have any suggestion how should look like dummy keypoints for padding?
